test/test_helper.exs

```diff
@@ -2,46 +2,65 @@ ExUnit.start()
 
 defmodule TestHelpers do
   use ExUnit.Case
-  import ExUnit.CaptureIO
+  use Plug.Test
 
-  def assert_console_output_matches(regex, action) do
-    assert Regex.match?(regex, capture_io(action)) 
+  @options TicTacToe.Web.Router.init([])
+
+  def get_request_with_session(path, session) do
+    request_with_session(:get, path, session)
   end
 
-  def get_cell_at(index, game) do
-    board = game.board
-    Enum.at(board.cells, index)
+  def post_request_with_session(path, session) do
+    request_with_session(:post, path, session)
+  end
+
+  defp request_with_session(method, path, session) do
+    conn(method, path)
+      |> init_test_session(session)
+      |> TicTacToe.Web.Router.call(@options)
+  end
+
+  def create_connection_with_session(session) do
+    conn(:get, "/") |> init_test_session(session)
   end
 
-  def create_human_player_with_moves([moves: moves, mark: mark, ui: ui]) do
-    moves_stream = create_input_stream(moves)
-    %Player.Human{io: {ui, moves_stream}, mark: mark}
+  def assert_response_includes_move_buttons(response, moves) do
+    Enum.each(moves, fn(move) ->
+      assert response.resp_body =~ move_button(move)
+    end)
   end
 
-  def create_human_player_with_moves([moves: moves, mark: mark]) do
-    moves_stream = create_input_stream(moves)
-    %Player.Human{io: {UI.Console, moves_stream}, mark: mark}
+  def move_button(move) do
+    "/tictactoe/moves/#{move}"
   end
 
-  def create_human_player_with_moves([moves: moves, ui: ui]) do
-    create_human_player_with_moves(moves: moves, mark: :x, ui: ui)
+#^^^^^^^^^^^^^^^^^^^^^^^^^^^ WEB ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

Comment:
Thanks for this, I now have a few tiny functions that do one thing rather than large ones that do more and are fragile! 

I also moved helpers into their separate files in the support folder.